inst/extdata/installRDependencies.sh

```diff
@@ -19,7 +19,8 @@ Rscript -e 'remove.packages("htmlTable")' || true
 
 echo '***** installing upward dependencies'
 Rscript -e 'install.packages(c("memoise","whisker","rstudioapi","roxygen2","dplyr","lubridate","rmarkdown"), repos="http://cran.us.r-project.org")'
-Rscript -e 'install.packages(c("httr","jsonlite","magrittr","knitr","htmltools","caTools","RCurl"), repos="http://cran.us.r-project.org")'
+Rscript -e 'install.packages("http://cran.r-project.org/src/contrib/Archive/httr/httr_1.0.0.tar.gz", repos=NULL, type="source")'
```

Comment:
Yeah, we have a jira ticket to do it. Going to let this evolve until right before rollout, then lock down.